.github/workflows/dokken-validate.yml

```diff
@@ -23,9 +23,19 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@main
+      - name: Get changed files
```

Comment:
nitpick, is there a way ti aviod the duplication of this action in the workflow?

Reply:
I couldn't find a solution quickly, we can improve in the future.